### ISSUE TITLE
Have read the userguid, have watched the video, but still dont know what the "schema migration system" is for - it needs a concrete high level overview.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [mybatis/old-google-code-issues#56]
```
I see something called "schema migration system". I read the user guide 
completely, but was left with more questions than were answered.  

E.g. I have a DB and schema creation scripts, why would I want to migrate it, 
and what does migration mean?  Migration from an existing schema into a mybatis 
managed one?  Can it handle grants/synonyms/tablespaces/user 
creation/indexes/partitioning/constraints/static data in respect to say Oracle? 


I watched the video, which only shows creation of tables on 2 diff empty DBs.  
Is this all it can do?  Does it manage changes to existing schemas?  Is the 
"changelog" table a user table, and part of this "blog" application, or is it a 
mybatis "system" table?  If so, should it not be prefixed with 
mybatis_changelog or similar, to avoid conflict?

In short, I could not tell if the complexity of using this migration system had 
any benifit over simply maintaining a single DB creation script, and say using 
toad to generate schema diff scripts to apply changes to production.

I think a better more concrete (less abstract) overview at the top fo the user 
guide would help.
```

Original issue reported on code.google.com by `hob...@gmail.com` on 23 Jun 2010 at 11:38
